Apps/CocoaPods-FCM/src/DashboardView.swift

```diff
@@ -0,0 +1,110 @@
+import CioTracking
+import SwiftUI
+
+struct DashboardView: View {
+    @State private var showingCustomEventSheet = false
+    @State private var showingCustomEventAlert = false
+    @State private var customEventName: String = ""
+    @State private var customEventPropertyName: String = ""
+    @State private var customEventPropertyValue: String = ""
+
+    @State private var showingDeviceAttributesSheet = false
+    @State private var showingDeviceAttributesAlert = false
+    @State private var deviceAttributeName: String = ""
+    @State private var deviceAttributeValue: String = ""
+
+    @State private var showingProfileAttributesSheet = false
+    @State private var showingProfileAttributesAlert = false
+    @State private var profileAttributeName: String = ""
+    @State private var profileAttributeValue: String = ""
+
+    var body: some View {
+        VStack(spacing: 40) {
+            Text("What would you like to test?")
+
+            ColorButton(title: "Send Random Event") {
+                CustomerIO.shared.track(
+                    name: String.random,
+                    data: [
+                        "randomAttribute": String.random,
+                        "random_attribute": String.random
+                    ]
+                )
+            }
+            ColorButton(title: "Send Custom Event") {
+                showingCustomEventSheet.toggle()
+            }.sheet(isPresented: $showingCustomEventSheet, content: {
+                VStack(spacing: 15) {
+                    TextField("Event name", text: $customEventName)
+                    TextField("Property name", text: $customEventPropertyName)
+                    TextField("Property value", text: $customEventPropertyValue)
+                    Button("Send event") {
+                        CustomerIO.shared.track(name: customEventName, data: [
```

Comment:
would there be any validation for these events? i remember adding them in RH, i think it would be good idea.